charts/virtaitech-orion-vgpu/v1.0.0/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
```

Comment:
Pandaria `2.3.x` branch using helm2 which not support `v2` API version.
The chart API version should be `v1`.

Reply:
Changed `apiVersion: v2` to `apiVersion: v1`